jmh-core-it/src/test/java/org/openjdk/jmh/it/fails/FailingBenchmarkBenchSetupTest.java

```diff
@@ -74,7 +74,7 @@ public void invokeAPI() throws RunnerException {
             new Runner(opt).run();
 
             Assert.fail("Should have failed");
```

Comment:
If you want to have fun, with JUnit 4.13.2 (see #24) you can use `assertThrows`. This code will become

```java
public void invokeAPI() throws RunnerException {
            Options opt = new OptionsBuilder()
                    .include(Fixtures.getTestMask(this.getClass()))
            RunnerException exception = assertThrows(() -> new Runner(opt).run());
            assertEquals(exception.getMessage(), "xxxxx"); // this is the bonus, checking it's the right message
}
```

Reply:
True, except that JMH codebase is still at JDK 7 compatibility, so no nice lambdas, and this gets much less appealing.

Reply:
Arg... Forgot that